src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -66,7 +66,7 @@ public function loadChoiceList($value = null)
             return $this->choiceList;
         }
 
-        return $this->choiceList = new ArrayChoiceList($this->getTimezones(), $value);
+        return $this->choiceList = new ArrayChoiceList(self::getTimezones(), $value);
```

Comment:
OH, but you can turn it protected, so still open for extension!
https://3v4l.org/f2STp